src/evo/creditpool.h

```diff
@@ -34,7 +34,7 @@ struct CCreditPool {
     // needs for logic of limits of unlocks
     CAmount currentLimit{0};
     CAmount latelyUnlocked{0};
-    CSkipSet indexes{};
+    CRangesSet indexes{};
```

Comment:
Note: CSkipSet is not used anywhere now, could probably drop it

Reply:
> Note: CSkipSet is not used anywhere now, could probably drop it
For sure, just thought to do it in a spare PR

Reply:
dead code should be dropped

Reply:
dead-code is removed, please, review @PastaPastaPasta